sys/vane/jael.hoon

```diff
@@ -889,6 +889,22 @@
     ::
         $vine
       +>(yen (~(put in yen) hen))
+    ::
+        %wegh
+      %_    +>
+          moz
+        :_  moz
+        ^-  move
+        :^  hen  %give  %mass
+        ^-  mass
+        :+  %jael  %|
+        :~  yen+&+yen
+            urb+&+urb
+            sub+&+sub
+            etn+&+etn
+            sap+&+sap
```

Comment:
A pattern we use in some vanes that I think we should use everywhere is to have an entry for the entire state (`lex` in this case) so that if we add something later it gets caught somewhere rather than falling through.  I think it would get caught in the `dot` entry in `++feck` in arvo, but I'm not sure.

Checking clay, for example, I think we're dropping a number of things that were added after the `%wegh` implementation because we don't use this pattern.

Reply:
I'll add `dot` tags to all the vanes, then. I feel like that's innocuous enough not to warrant a separate PR.

Reply:
done